crates/sui-protocol-config/src/lib.rs

```diff
@@ -2999,6 +3004,8 @@ impl ProtocolConfig {
                         // Enable probing for accepted rounds in round prober for testnet
                         cfg.feature_flags
                             .consensus_round_prober_probe_accepted_rounds = true;
+                        // [SIP-45] Set max gas price to 1T.
+                        cfg.max_gas_price = Some(1_000_000_000_000);
```

Comment:
Protocol version 70 is already released and we cannot modify it. We need to set `max_gas_price` in the next protocol version 71.